### DISCUSSION TITLE
gatsby-cli install hanging in windows admin terminal

### DISCUSSION THREAD

**Discussion by robwelan** [gatsbyjs/gatsby#32168] [Help]
In an admin terminal on Windows 10 (build 19042.1052) I am performing an npm install -g gatsby-cli.

It consistently hangs on this line:
 / reify:prettier: http fetch GET 200 https://registry.npmjs.org/prettier/-/prettier-2.3.2.tgz 2763

how to fix?

NOTE: I have installed the latest node with extras which is v14.17.1-x64, and I have done an npm cache clean --force.

**Comment by LekoArts** [comment]
Hi!

This isn't a specific issue with Gatsby but with npm and your local setup. You might try the suggestions here: https://github.com/npm/npm/issues/13729
